docs/swagger.yaml

```diff
@@ -3,7 +3,7 @@ info:
   title: Open edX API
   description: APIs for access to Open edX information
   contact:
-    email: oscm@edx.org
+    email: api-support@edx.org
```

Comment:
I'd rather have this email come from settings, but in lieu of that, the [spec](https://swagger.io/specification/) says that we can just omit the `contact` field entirely. How does that sound?

Reply:
Done.

Reply:
Another place: https://github.com/openedx/api-doc-tools/blob/master/edx_api_doc_tools/conf_utils.py#L41-L46

Looks like there is more to do:
```
  File "/home/runner/work/edx-platform/edx-platform/lms/urls.py", line 37, in <module>
    from openedx.core.apidocs import api_info
  File "/home/runner/work/edx-platform/edx-platform/openedx/core/apidocs.py", line 9, in <module>
    api_info = make_api_info(
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/edx_api_doc_tools/conf_utils.py", line 61, in make_api_info
    contact=openapi.Contact(email=email),
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/drf_yasg/openapi.py", line 184, in __init__
    raise AssertionError("one of name, url or email is requires for Swagger Contact object")
AssertionError: one of name, url or email is requires for Swagger Contact object
```